configs/default/HENA-TALONF7FUSION.config

```diff
@@ -80,7 +80,7 @@ serial 0 2048 115200 57600 0 115200
 serial 2 64 115200 57600 0 115200
 
 # aux
-aux 0 39 0 900 2100 0 0
+aux 0 40 0 900 2100 0 0
```

Comment:
I agree that changing the mode controlling the VTX power pin is something that _should_ be changed, as the existing setup _can_ lead to problems if users use a VTX that is controlled through Betaflight.
However, we need to make sure that the manufacturers of the hardware in question are in on this, and **change their documentation to reflect this change**. If not, we are changing something that will not work for _some_ users of this feature to something that will not work for _all_ users of this feature.
I think @bnn1044 is representing the manufacturer for this board?